### ISSUE TITLE
Support adding new types of nodes and edges to an existing property graph

### ISSUE THREAD

**Issue by sighingnow** [alibaba/GraphScope#138]
**Is your feature request related to a problem? Please describe.**

We store a CSRs for per `(edge_label, src_label, dst_label)` tuple, adding a new kinds of edges should be possible.

Add new kind of nodes might be a bit tricker since it is related to encoding of `GID`s.

Note that for adding new kind of edges, it means adding a new `edge_label`, rather than adding a new `(src_label, dst_label)` pair for an existing `edge_label`.

**Describe the solution you'd like**

Add API like `Graph.add_edge()` and `Graph.add_node()`.

**Additional context**

See also the proposal for property fragment project: #134.

**Comment by siyuan0322** [comment]
Add Edges API proposal (feel free to comment):
```python
# suppose we already load a graph
 graph = load_graph(edges="xxx", vertices="xxx")

# Current load_from style loader.
 graph = graph.add_edges(
     {
         "label": [
             "path/to/file",
             ["prop1", "prop2"],
             ("src_label", "src_id"),
             ("dst_label", "dst_id"),
         ]
     }
 )

# Use a helper class
 class Edge:
     def __init__(self, label, src_label, dst_label, properties=None):
         label = label
         properties = [] if properties is None else properties
         src_label = src_label
         dst_label = dst_label


 e1 = Edge(label="l1", src_label="v1", dst_label="v2")
 e2 = Edge(label="l2", src_label="v3", dst_label="v4", properties=["p1", "p1"])

# Can add multiple edges at one time.
 graph = graph.add_edges(e1)
 graph = graph.add_edges([e1, e2, e3])
```

**Comment by siyuan0322** [comment]
Closed by #179
